src/parsers/manifestjson.js

```diff
@@ -392,6 +394,87 @@ export default class ManifestJSONParser extends JSONParser {
     return Promise.all(promises);
   }
 
+  validateStaticThemeImages() {
+    const promises = [];
+    const themeImages = this.parsedJSON.theme && this.parsedJSON.theme.images;
+
+    const validateThemeImage = async (imagePath, manifestPropName) => {
+      const _path = normalizePath(imagePath);
+      const ext = path
+        .extname(imagePath)
+        .substring(1)
+        .toLowerCase();
+
+      const fileExists = this.validateFileExistsInPackage(
+        _path,
+        `theme.images.${manifestPropName}`,
+        messages.manifestThemeImageMissing
+      );
+
+      // No need to validate the image format if the file doesn't exist
+      // on disk.
+      if (!fileExists) {
+        return;
+      }
+
+      if (!IMAGE_FILE_EXTENSIONS.includes(ext) || ext === 'webp') {
+        this.collector.addError(
+          messages.manifestThemeImageWrongExtension({ path: _path })
+        );
+        this.isValid = false;
+        return;
+      }
+
+      try {
+        const info = await getImageMetadata(this.io, _path);
+        if (
```

Comment:
This is definitely a style thing, so up to you, but I'd expect this to be more like:

```js
if (!STATIC_THEME_IMAGE_MIMES.includes(info.mime)
  || info.mime === 'image/webp') {
```

Reply:
eh... to be honest I was aiming for something like that, but prettier strongly disagrees :-)
we have been fighting a bit each other (prettier and I) but it quickly started to look as a "quite pointless battle"  and I decided that I can live happy enough with the version that makes "prettier" happy :-)